max_speed: 4 # vitesse maxAckermann m/s 
max_steer: 0.36 # max en radian     
dist_critique: 2 # distance avant de commencer a ralentir
dist_secu: 0.3 #distance des obstacles

#Valeur max et min du laser pour le calcul de la distance max
min_id: 212
max_id: 812
        
id_decalage: 80 #decalage si obstacles
distance_obst: 1.5 #rayon des points pris en compte pour le calcul obstacle
angle_coef: 0.9 #coef entre l'angle cible et l'angle de consigne

# Joystick indices for toggling mux
auto_button_idx: 5 # R1 button  
